Code/Engine/RendererCore/AnimationSystem/AnimGraph/Implementation/AnimGraphPins.cpp

```diff
@@ -51,12 +51,8 @@ void ezAnimGraphTriggerOutputPin::SetTriggered(ezAnimGraph& ref_graph, bool bTri
   if (m_iPinIndex < 0)
     return;
 
-  if (!bTriggered)
```

Comment:
Looking at the rest of the logic, the current code is correct, and this change would break it.
There was a design change at some point, which meant that this code works differently now than originally.

So to get rid of the warning, you can change this line:
const ezInt8 offset = +1; // bTriggered ? +1 : -1;

but keep the if (!bTriggered) return;

It is unclear whether this piece of code will stay that way in the long run, but for now that's good enough.